app/services/push_notification/http_push.rb

```diff
@@ -38,72 +38,23 @@ def url_options
     def deliver_one(service_provider)
       deliver_local(service_provider) if IdentityConfig.store.risc_notifications_local_enabled
 
-      if IdentityConfig.store.risc_notifications_eventbridge_enabled
-        deliver_eventbridge(service_provider)
-      else
-        deliver_direct(service_provider)
-      end
-    end
-
-    def deliver_eventbridge(service_provider)
-      response = eventbridge_client.put_events(
-        entries: [
-          {
-            time: now,
-            source: service_provider.issuer,
-            detail_type: 'notification',
-            detail: { jwt: jwt(service_provider) }.to_json,
-            event_bus_name: "#{Identity::Hostdata.env}-risc-notifications",
-          },
-        ],
-      )
-
-      if response.failed_entry_count.to_i > 0
-        Rails.logger.warn(
-          {
-            event: 'http_push_error',
-            transport: 'eventbridge',
-            event_type: event.event_type,
-            service_provider: service_provider.issuer,
-            error: response.to_s,
-          }.to_json,
+      if IdentityConfig.store.risc_notifications_active_job_enabled
+        RiscDeliveryJob.perform_later(
+          push_notification_url: service_provider.push_notification_url,
+          jwt: jwt(service_provider),
+          event_type: event.event_type,
+          issuer: service_provider.issuer,
+          transport: 'ruby_worker',
```

Comment:
if there was a way to infer if a job was performed inline vs in a thread, I'd be happy to use that instead of passing a string like this